ci/Makefile

```diff
@@ -56,21 +54,17 @@ push: build
 	docker tag ci $(CI_IMAGE)
 	docker push $(CI_IMAGE)
 
-deploy: push push-builder-utils
-	python3 ../builder/jinja2_render.py '{"code":{"sha":"$(shell git rev-parse --short=12 HEAD)"},"deploy":true,"default_ns":{"name":"default"},"ci_image":{"image":"$(CI_IMAGE)"},"global":{"project":"$(PROJECT)","ip":"$(IP)","domain":"$(DOMAIN)"},"builder_utils_image":{"image":"$(BUILDER_UTILS_IMAGE)"},"ci_database":{"user_secret_name":"sql-ci-ci-user-config"}}' deployment.yaml deployment.yaml.out
+deploy: push push-ci-utils
+	python3 jinja2_render.py '{"code":{"sha":"$(shell git rev-parse --short=12 HEAD)"},"deploy":true,"default_ns":{"name":"default"},"ci_image":{"image":"$(CI_IMAGE)"},"global":{"project":"$(PROJECT)","ip":"$(IP)","domain":"$(DOMAIN)"},"ci_utils_image":{"image":"$(CI_UTILS_IMAGE)"},"ci_database":{"user_secret_name":"sql-ci-ci-user-config"}}' deployment.yaml deployment.yaml.out
 	kubectl -n default apply -f deployment.yaml.out
 else
-build-builder-utils:
+build-ci-utils:
 	make -C ../docker build
-	make -C ../builder \
-	  BUILDER_UTILS_LATEST="$(BUILDER_UTILS_LATEST)" \
-          BUILDER_UTILS_IMAGE="$(BUILDER_UTILS_IMAGE)" \
-          IN_HAIL_CI="$(IN_HAIL_CI)" \
-          build
+	docker build -t ci-utils -f Dockerfile.ci-utils .
 
 build:
 	make -C ../docker build
-	docker build -t ci -f Dockerfile ..
+	docker build -t ci -f Dockerfile.ci ..
```

Comment:
[Dockerfile wasn't renamed to Dockerfile.ci](https://github.com/hail-is/hail/tree/3caadcc6000883d88847777a5a1626d21815bf06/ci0

Reply:
Good catch. That was a bug in the original code. Fixed.